cmake/projects/Eigen/hunter.cmake

```diff
@@ -17,26 +17,13 @@ hunter_add_version(
     PACKAGE_NAME
     Eigen
     VERSION
-    "3.2.4-p0"
+    "3.3.1"
     URL
-    "https://github.com/hunter-packages/eigen/archive/v3.2.4-p0.tar.gz"
+    "http://bitbucket.org/eigen/eigen/get/3.3.1.tar.bz2"
     SHA1
-    6fd638939dfd3a3be1cf0ea920160591ed60f76d
+    d1a9c2b64db3628956482b4a934cab3b508dc29e
 )
 
-hunter_add_version(
-    PACKAGE_NAME
-    Eigen
-    VERSION
-    "3.2.4"
-    URL
-    "https://bitbucket.org/eigen/eigen/get/3.2.4.tar.gz"
-    SHA1
-    b826f665bdfe31784795eef79c75540db71ab702
-)
-
-hunter_cmake_args(Eigen CMAKE_ARGS EIGEN_ENABLE_TESTING=OFF)
```

Comment:
> Also as you may noticed in comment, this variable fix the Android build hunter-packages/eigen@930d0a2

could you test what exactly breaks the Android build. I havn't built for Android. Maybe we can send a patch upstream to the Eigen developers which disables tests